tests/unit/Mage/Core/Helper/DataTest.php

```diff
@@ -93,9 +93,9 @@ public function testFormatTimezoneDate(
     public function provideFormatTimezoneDate(): Generator
     {
         $date           = date_create()->getTimestamp();
-        $dateShort      = date('m/j/Y', $date);
+        $dateShort      = date('n/j/Y', $date);
```

Comment:
Can explain why remove leading zero in day?

Reply:
> --- Expected
+++ Actual
@@ @@
-'01/5/2025'
+'1/5/2025'

"formatTimezoneDate()" seems not to have a leading zero.